tests/integration/devfile/cmd_devfile_registry_test.go

```diff
@@ -48,6 +48,11 @@ var _ = Describe("odo devfile registry command tests", func() {
 			helper.MatchAllInOutput(output, []string{"DefaultDevfileRegistry"})
 		})
 
+		It("Should list all default registries with json", func() {
+			output := helper.CmdShouldPass("odo", "registry", "list", "-o", "json")
+			helper.MatchAllInOutput(output, []string{"DefaultDevfileRegistry"})
```

Comment:
please add a check to verify that the output is in json format. This might hep you https://stackoverflow.com/questions/22128282/how-to-check-string-is-in-json-format